.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules/
```

Comment:
`node_modules` should do. For directories just put the directory name without the back n forward slashes.